language/move-lang/src/expansion/translate.rs

```diff
@@ -1184,8 +1238,11 @@ fn unbound_names_assigns(unbound: &mut BTreeSet<Name>, sp!(_, ls_): &E::LValueLi
 fn unbound_names_assign(unbound: &mut BTreeSet<Name>, sp!(_, l_): &E::LValue) {
     use E::LValue_ as EL;
     match l_ {
-        EL::Var(v) => {
-            unbound.insert(v.0.clone());
+        EL::Var(en, _) => {
+            // Qualified vars are not considered in unbound set.
+            if let E::ModuleAccess_::Name(n) = &en.value {
+                unbound.insert(n.clone());
+            }
```

Comment:
Same as `unbound_names_bind`, lift this to the outer match

Reply:
Done.